app/assets/stylesheets/admin/pages/login/_structures.scss

```
@@ -13,3 +13,10 @@
     top: 258px;
   }
 }
+
+.new {
+  .bouton-retour {
+    position: absolute;
+    left: -197px;
```

Il est bizarre ce chiffre, je pense qu'il vaut mieux faire : 

```suggestion
    right: 100%;
```